docs/release_notes.md

```diff
@@ -1,5 +1,34 @@
 # Release Notes (What's New)
 
+## Version 1.1.0 (Upcoming Release) 
+
+For the full details of all changes in this release, see the [GitHub commit history](https://github.com/nci/scores/compare/1.0.0...develop). Below are the changes we think users may wish to be aware of.
+
+### Features
+
+- Added five new threshold weighted scores 
+	- threshold weighted squared error: ` scores.continuous.tw_squared_error`
+	- threshold weighted absolute error: `scores.continuous.tw_absolute_error`
+	- threshold weighted quantile score: `scores.continuous.tw_quantile_score`
+	- threshold weighted expectile score: `scores.continuous.tw_expectile_score`
+	- threshold weighted huber loss: `scores.continuous.tw_huber_loss`.  
+See [PR #609](https://github.com/nci/scores/pull/609) by [@nicholasloveday](https://github.com/nicholasloveday).
+
+### Breaking Changes
+
+### Deprecations
+
+### Bug Fixes
+
+### Documentation
+
+- Added "Threshold Weighted Scores" tutorial. See [PR #609](https://github.com/nci/scores/pull/609) by [@nicholasloveday](https://github.com/nicholasloveday).
```

Comment:
I didn't originally think to include the new "Threshold Weighted Scores" tutorial as it's own entry, as it was part of the larger pull request for the new threshold weighted scores (which was already mentioned in the "Features" section). However, I think the addition of a new tutorial is noteworthy, and likely to be of interest to users. So I thought it was perhaps worth adding specifically.

@nicholasloveday did you want any additional detail to be added?

Reply:
Looks good thanks